compiler/src/iree/compiler/Codegen/Common/MaterializeUserConfigs.cpp

```diff
@@ -40,6 +40,31 @@ namespace {
 
 static const char kTranslationInfoAttrName[] = "translation_info";
 
+/// Sets compilation configuration annotated in the incoming IR.
+LogicalResult
```

Comment:
nit: make static

Reply:
well, its already in anonymous namespace

Reply:
Oh that's true. If it means rerunning CI don't worry about it, but I find having the static there helpful for reading the code.